atlasaction/git/conflict.go

```diff
@@ -5,7 +5,7 @@ import (
 	"strings"
 )
 
-var filenameRegex = regexp.MustCompile(`\b\d+_[\w-]+\.sql\b`)
+var filenameRegex = regexp.MustCompile(`\b\d+(?:_[\w-]+)?\.sql\b`)
```

Comment:
before if file name was `2025030900000.sql` and not `2025030900000_some_name.sql`, the regex didnt recognized it

Reply:
Why you don't use `path.Base()` or `path.Extension` to check the file's extension?

Reply:
Also, `migrate.HashFile` can be use for unmmarshal the `atlas.sum`